src/validationplots/plots.jl

```diff
@@ -18,12 +37,68 @@ end
 """
 # TODO : desc
 """
-function probplot(fm::MaximumLikelihoodEVA)::Plot
+function probplot_data(fm::BayesianEVA)::DataFrame
+
+    checkstationarity(fm.model)
+
+    y, p̂ = ecdf(fm.model.data.value)
+
+    dist = getdistribution(fm)
+
+    p_sim = Array{Float64}(undef, length(dist), length(y))
+
+    i = 1
+
+    for d in eachslice(dist, dims=1)
+        p_sim[i,:] = cdf.(d, y)
+        i +=1
+    end
 
-    df = probplot_data(fm)
+    p = vec(mean(p_sim, dims=1))
+
+    return DataFrame(Model = p, Empirical = p̂)
+
+end
+
+
+
+"""
+# TODO : desc
+"""
+function qqplot(fm::fittedEVA)::Plot
+
+    if getcovariatenumber(fm.model) > 0
+        df = qqplot_std_data(fm)
+        plotTitle = "Residual Quantile Plot"
+    else
+        df = qqplot_data(fm)
+        plotTitle = "Quantile Plot"
+    end
 
     return plot(df, x=:Model, y=:Empirical, Geom.point, Geom.abline(color="red", style=:dash),
-        Guide.xlabel("Model"), Guide.ylabel("Empirical"), Guide.title("Probability Plot"))
+        Guide.xlabel("Model"), Guide.ylabel("Empirical"), Guide.title(plotTitle))
+
+end
+
+
+"""
+# TODO : desc
+"""
+function qqplot_data(fm::pwmEVA)::DataFrame
```

Comment:
Here I am being picky and you can ignore this if you don't think it necessary! 😄 
Every time we have functions related to the `fittedEVA` structures, the order is as follow: `bayesianEVA`, `maximumlikelihoodEVA` and then `pwmEVA` (alphabetical). The data functions here do not follow this order.

Reply:
Done